src/app/components/result-box/result-box.component.ts

```diff
@@ -9,16 +9,22 @@ export class ResultBoxComponent {
   @Input() word = '';
   @Input() splitWord: string[] = [];
   @Input() borderResultStyles!: boolean;
+  @Input() keyBoxStyle: string = '';
   @Input() firstRound = false;
   @Input() selectedBox!: number;
   @Output() selectResultBox = new EventEmitter<number>();
 
-  borderStyles(): string {
-    if (!this.firstRound) return '';
-    return this.borderResultStyles ? 'border-success' : 'border-danger';
-  }
-
   selectedResultBox(boxIndex: number) {
     this.selectResultBox.emit(boxIndex);
   }
+
+  newBoxStyles(letter: string, index: number) {
+    return {
+      '': !this.firstRound,
+      hit: this.keyBoxStyle === 'hit',
+      parcialHit: this.keyBoxStyle === 'parcialHit',
+      fail: this.keyBoxStyle === 'fail',
+      'border-primary': this.selectedBox === index,
```

Comment:
```suggestion
      'selectedSquare': this.selectedBox === index,
```

Use a more clarifing variable

Reply:
es de boostrap, si pongo el selectedSquare no me deja poner un borde con color de normal dentro de css